discretisedfield/field.py

```diff
@@ -2834,18 +2838,73 @@ def _writeovf(self, filename, representation='bin8', extend_scalar=False):
 
             f.write(bfooter)
 
-    def _writevtk(self, filename):
+    def _to_vtk(self):
```

Comment:
This function could be potentially useful in the future. As we can now easily convert a field to vtk we could think about adding support for interactive 3d plots based on e.g. pyvista ([demo](https://docs.pyvista.org/user-guide/jupyter/index.html)) in addition to `k3d`. That could also just be a notebook demonstrating how to do it. So instead of hiding it or moving to `util` would rather think about making it public right away.

Reply:
Sure, sounds good. I was just worried giving users too many options to read/save the file.

Reply:
Shall we then make it public right away?